### DISCUSSION TITLE
RegistryManager still uses a default timeout of 100 seconds.

### DISCUSSION THREAD

**Discussion by itziksaban** [Azure/azure-iot-sdk-csharp#2480] [General]
Hi everyone,
I saw that in release 1.37.1 you removed the default timeout from ServiceClient. Unfortunately, we are using RegistryManager to GetDeviceAsync, and there, it seems like the default timeout is still 100 seconds. Any plans to change that as well?

Thanks
Itzik

**Comment by timtay-microsoft** [comment]
RegistryManager operations (including GetDeviceAsync) all allow you to pass in a cancellation token in case you want to set a custom timeout for each operation. With that in mind, is there some reason why you'd still like the default timeout to be changed?

**Reply by itziksaban** [reply]
Thank you timtay, I overlooked that option.
